tools/model_explorer_circle/pyproject.toml

```diff
@@ -0,0 +1,7 @@
+[project]
```

Comment:
I think we can split this file to another PR

Reply:
I think it is ok with `.gitignore` as initial file.
As I know, `pyproject.toml` is for publication so we can add this file for that purpose.